plugins/outputs/timestream/timestream.go

```diff
@@ -373,35 +470,36 @@ func (t *Timestream) createTable(tableName *string) error {
 // Telegraf Metrics are grouped by Name, Tag Keys and Time to use Timestream CommonAttributes.
 // Returns collection of write requests to be performed to Timestream.
 func (t *Timestream) TransformMetrics(metrics []telegraf.Metric) []*timestreamwrite.WriteRecordsInput {
-	writeRequests := make(map[uint64]*timestreamwrite.WriteRecordsInput, len(metrics))
+	writeRequests := make(map[string]*timestreamwrite.WriteRecordsInput, len(metrics))
+
 	for _, m := range metrics {
 		// build MeasureName, MeasureValue, MeasureValueType
+
 		records := t.buildWriteRecords(m)
+
 		if len(records) == 0 {
 			continue
 		}
-		id := hashFromMetricTimeNameTagKeys(m)
```

Comment:
looks like you can also remove the hashFromMetricTimeNameTagKeys function. I'm assuming grouping them by tag and timestamp was not helpful.

Reply:
Removed

Reply:
Sounds like a plan!

Reply:
Removed in the latest commit